values.yaml

```
@@ -450,6 +450,8 @@ rabbitmq-ha:
   # Change to `false` to disable in-cluster rabbitmq deployment.
   # Specify your external [messaging] connection parameters under st2.config
   enabled: true
+  # On unclean cluster restarts forceBoot is required to cleanup Mnesia tables (see: https://github.com/helm/charts/issues/13485)
+  forceBoot: true
```

As every option is about trade-off, worth adding comment for this new default:

```suggestion
  # On unclean cluster restarts forceBoot is required to cleanup Mnesia tables (see: https://github.com/helm/charts/issues/13485)
  # Use it only if you prefer availability over integrity.
  forceBoot: true
```

---

Suggested comment is added